lib/core/test/unit/Cardano/Wallet/Primitive/AddressDiscovery/SequentialSpec.hs

```diff
@@ -652,3 +647,7 @@ data Key = forall (k :: Depth -> * -> *).
     , AddressPoolTest k
     ) => Key (Proxy k)
 instance Show Key where show (Key proxy) = show (typeRep proxy)
+
+dummyMnemonic :: SomeMnemonic
+dummyMnemonic =
+    unsafeMkSomeMnemonicFromEntropy (Proxy @12) (BS.pack $ replicate 16 0)
```

Comment:
Replacing both 16- and 32- bytes long entropies in this module with 16, seems iffy.

Adding a `dummyMnemonic :: (many constraints) => Proxy mw -> SomeMnemonic` sounds tempting…

But in-doubt, I will not, I think.

Will remove `dummyMnemonic`.